modules/verify-knative-serving-monitoring.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/monitoring-serverless.adoc
+
+[id="configuring-knative-services-domains_{context}"]
+= Verifying an {product-title} monitoring installation for use with Knative Serving
+
+Manual configuration for monitoring by an administrator is not required, but you can carry out these steps to verify that monitoring is installed correctly.
+
+.Procedure
+
+. Verify that the ServiceMonitor objects are deployed.
++
+----
+$ oc get servicemonitor -n knative-serving
+NAME         AGE
+activator    11m
+autoscaler   11m
+controller   11m
+----
++
```

Comment:
One note here - this `+` is not necessary, because we're proceeding to the next element in the ordered list. Right now it's not causing an issue, as these instructions render correctly. Just giving you a heads up for future procedures. :)